src/main/java/nextstep/domain/Ladder.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ladder {
+    Lines lines;
+    Participants participants;
```

Comment:
```suggestion
    private final Lines lines;
    private final Participants participants;
```
인스턴스 변수는 가능하면 private final로 구현할 것을 추천